test/test_getentity.py

```diff
@@ -42,6 +42,11 @@ def testGetArtist(self):
                 musicbrainzngs.get_artist_by_id,
                 artistid, ["release-groups"], release_status=["official"])
 
+    def testGetEvent(self):
+        event_id = "a4a0927c-8ad7-48dd-883c-7126cc0b9c6b"
+        musicbrainzngs.get_event_by_id(event_id)
+        self.assertEqual("http://musicbrainz.org/ws/2/event/a4a0927c-8ad7-48dd-883c-7126cc0b9c6b", self.opener.get_url())
+
```

Comment:
I'd like to see some tests here with various includes and release type/status to validate the validation (ha)